gen2-sample/csharp-tsi-gen2-sample/DataPlaneClient/Readme.md

```diff
@@ -0,0 +1,7 @@
+# Time Series Insights (Gen2 GA) - Data Plane Client
```

Comment:
Call it "Azure Time Series Insights (Gen2)" for consistency. Have we been using the term Gen2 GA consistently everywhere?